content/docs/command-reference/fetch.md

```diff
@@ -12,7 +12,7 @@ usage: dvc fetch [-h] [-q | -v] [-j <number>] [-r <name>] [-a] [-T]
 
 positional arguments:
   targets       Limit command scope to these tracked files/directories,
-                .dvc files, or stage names.
+                stages, or .dvc files.
```

Comment:
Mention stages before .dvc files (as in other definitions).

- [ ] This technically needs a core repo update...

Reply:
What other definitions? Looking at the core help text, I just checked `push`, `pull`, and `checkout`, and all had `files/directories, .dvc files, or stage names`.

Reply:
@jorgeorpinel This is the only question I have left. Is this change actually needed? Where did it come from?

Reply:
This particular change in the Synopsis is not needed @dberenbaum . It's a marginal improvement IMO (stage names are more important than .dvc files).

Reply:
Would you mind if we take this change out? I don't think it's worth it because:
1. It requires a core change
2. All the similar commands seem to use the current order
3. It makes more sense to me in the current order because the first two are files.

Reply:
Taken out for now!

Reply:
> What other definitions? ... `push`, `pull`, and `checkout` all have "tracked files/directories, .dvc files, or stage names".

Yeah you're right, not sure why I thought we'd changed those before. Also `status`

> makes more sense to me in the current order because the first two are files

The first one is "tracked files/dirs" so not exactly files and ".dvc files" in that sense is more of a configuration item. That is to say that to my mind they're in different buckets anyway. So stage names still seems more important to me (marginally), but I'll leave it up to you to consider whether to make an issue about it in the core repo.

Thanks again @dberenbaum